src/commands/_shared.ts

```diff
@@ -1,32 +1,47 @@
-export const sharedArgs = {
+import type { ArgDef } from 'citty'
+
+export const cwdArgs = {
   cwd: {
     type: 'string',
-    description: 'Current working directory',
+    description: 'Specify the working directory',
+    valueHint: 'directory',
+    default: '.',
   },
+} as const satisfies Record<string, ArgDef>
+
+export const logLevelArgs = {
   logLevel: {
     type: 'string',
-    description: 'Log level',
+    description: 'Specify build-time log level',
+    valueHint: 'silent|info|verbose',
   },
-} as const
+} as const satisfies Record<string, ArgDef>
 
 export const envNameArgs = {
   envName: {
     type: 'string',
     description: 'The environment to use when resolving configuration overrides (default is `production` when building, and `development` when running the dev server)',
   },
-} as const
+} as const satisfies Record<string, ArgDef>
 
 export const dotEnvArgs = {
   dotenv: {
     type: 'string',
-    description: 'Path to .env file',
+    description: 'Path to `.env` file to load, relative to the root directory',
   },
-} as const
+} as const satisfies Record<string, ArgDef>
 
 export const legacyRootDirArgs = {
+  // cwd falls back to rootDir's default (indirect default) to ease migration
+  cwd: {
+    ...cwdArgs.cwd,
+    description: 'Specify the working directory (default: `.`)',
+    default: undefined,
+  },
   rootDir: {
     type: 'positional',
-    description: 'Root Directory',
+    description: '(DEPRECATED) Use `--cwd` instead. Specifies the working directory (default: `.`)',
```

Comment:
hm.... my understanding was that the positional rootdir wasn't deprecated, it's the argument `--rootDir` that is deprecated.

the ROOTDIR here would be something like:

```bash
npx nuxi dev nuxt-app
```

but maybe I'm wrong - @pi0 were you thinking to deprecate the positional rootDir?

Reply:
All I know is that `rootDir` was marked as legacy and that `---cwd` takes precedence over its usage since https://github.com/nuxt/cli/pull/47, while it wasn't explicitly marked as deprecated, dropping the argument was discussed in https://github.com/nuxt/cli/issues/450. 

I guess the main advantage of `--cwd` is that it isn't positional, but I can imagine there's plenty of scripts that rely on passing a `rootDir` directory without flag.

Reply:
Let's not deprecate the positional rootDir for now. 🙏

(I think the thing that's deprecated is the _name_ `rootDir` - but we can follow up on that in a separate PR.)

Reply:
Yep, that was purpose of deprecation rootDir -> cwd by name. 

I wish we never had positional dir but that ship already sailed for nuxi main commands.

Reply:
I see, I have removed the deprecation description now, but did add a description that `cwd` argument takes precedence over `rootDir` for clarity.

Reply:
perfect! ❤️